public/index.html

```diff
@@ -49,8 +49,14 @@ <h4> Create a Geolocating Web Annotation </h4>
                 </a>
                 
             </div>
+	    <footer>
+		    <a class="open source" href="https://github.com/CenterForDigitalHumanities/geolocator/tree/main">
+			    This is open source. Improve this page!
```

Comment:
This is what we want, but you should define a new custom element, then add that element to the layout.  This will have the effect of getting it onto every page.

See `GeoPage`, `GeoHeader`, `GeoFooter` in [layout.js](https://github.com/CenterForDigitalHumanities/geolocator/blob/44-open-source-statement/public/components/layout.js#L10).  This is where your code changes should be.  Either make the notice a part of the existing header/footer, or create your own `OSnotice` element to add to `GeoPage` like the header and footer are.